### DISCUSSION TITLE
how to check if pipes are enabled (or not)

### DISCUSSION THREAD

**Discussion by gviznyuk** [ImageMagick/ImageMagick#7539] [General]
What is the command line to check if pipes are enabled (or not) for this library.  This is related to https://github.com/ImageMagick/ImageMagick/issues/6339

**Comment by dlemstra** [comment]
There is no command line option to check this. If the include files that were used for the build are available on your system you can check if `MagickCore/magick-baseconfig.h` contains `MAGICKCORE_PIPES_SUPPORT`.
